tempesta_fw/http_parser.c

```diff
@@ -1495,7 +1470,41 @@ tfw_http_parse_req(void *req_data, unsigned char *data, size_t len)
 		}
 	}
 
-	RGEN_LWS();
+	__FSM_STATE(RGen_LWS) {
```

Comment:
What was the reason that required the removal of `RGen_LWS()` macro, and moving its whole contents to the body of this function, as well as to the body of `tfw_http_parse_resp()`?